src/components/chip/usage/Basic.md

```diff
@@ -1,3 +1,3 @@
 ```html
-<calcite-chip value="Global" dismissible icon="globe" appearance="clear" color="green">Global</calcite-chip>
+<calcite-chip value="Global" closable icon="globe" appearance="clear" color="green">Global</calcite-chip>
```

Comment:
@anveshmekala one more thing...

I think the `appearance` value of `clear` is deprecated. Can we change them all to `transparent`?

Reply:
We could always do this later if you would like that better.

Reply:
Good catch. will change it to `transparent`. And get the `conditionalSlotComponent` removed as well.

Reply:
We can do the `connectConditionalSlotComponent` in a separate pr/epic. Lets wait on that one.